evm/tests/transfer_to_new_addr.rs

```diff
@@ -0,0 +1,46 @@
+use hex_literal::hex;
+use plonky2::field::goldilocks_field::GoldilocksField;
+use plonky2::plonk::config::PoseidonGoldilocksConfig;
+use plonky2::util::timing::TimingTree;
+use plonky2_evm::all_stark::AllStark;
+use plonky2_evm::config::StarkConfig;
+use plonky2_evm::generation::{generate_traces, TransactionData};
+use plonky2_evm::prover::prove;
+use plonky2_evm::verifier::verify_proof;
+
+type F = GoldilocksField;
+const D: usize = 2;
+type C = PoseidonGoldilocksConfig;
+
+/// Test a simple token transfer to a new address.
+#[test]
+#[ignore] // TODO: Won't work until txn parsing, storage, etc. are implemented.
+fn test_simple_transfer() -> anyhow::Result<()> {
```

Comment:
Added this test to illustrate what we're working toward. Once we get storage etc working, we can add more realistic tests like this, and eventually delete the "fake" test stuff like `make_cpu_trace`, `generate_random_memory_ops`, etc.